### PR TITLE
TRAFODION-2686 fix setCurSorName does not work

### PR DESCRIPTION
**Pull request by luoyunpeng**

### CODE REVIEW

**Comment by luoyunpeng** [comment]
@kevinxu021 Hi kevin please help review this change

**Comment by svarnau** [comment]
jenkins, ok to test

**Comment by luoyunpeng** [comment]
the failure does not belong to jdbcT4,

**Comment by luoyunpeng** [comment]
jenkins, retest

**Comment by kevinxu021** [comment]
Jenkins, retest

File: core/conn/jdbcT4/src/main/java/org/trafodion/jdbc/t4/TrafT4Statement.java [hunk 1]
```diff
@@ -1247,7 +1247,7 @@ void setMultipleResultSets(int num_result_sets, TrafT4Desc[][] output_descriptor
 			if (desc == null) {
 				resultSet_[i] = null;
 			} else {
-				resultSet_[i] = new TrafT4ResultSet(this, desc, stmt_labels[i], true);
+				resultSet_[i] = new TrafT4ResultSet(this, desc, stmt_labels[i], this.sqlStmtType_==TRANSPORT.TYPE_CALL);
```
Comment (selvaganesang) [inline core/conn/jdbcT4/src/main/java/org/trafodion/jdbc/t4/TrafT4Statement.java]: The last parameter  seems to denote oldDataFormat in TrafT4ResultSet.java.  Can you please provide some explanation about this change?

Comment (luoyunpeng) [inline core/conn/jdbcT4/src/main/java/org/trafodion/jdbc/t4/TrafT4Statement.java]: this is the parameter pass to spj_rs which is always true  that cause setCursorName in TraT4Statement.java does not work,

Comment (selvaganesang) [inline core/conn/jdbcT4/src/main/java/org/trafodion/jdbc/t4/TrafT4Statement.java]: Thanks for the explanation.  Read the surrounding the code and it made sense now

Comment (luoyunpeng) [inline core/conn/jdbcT4/src/main/java/org/trafodion/jdbc/t4/TrafT4Statement.java]: Thanks for helping review,

**Pull request merged**